.github/workflows/bot-xrp-mere-denis.yml

```diff
@@ -0,0 +1,79 @@
+name: Bot 'XRP on Mère Denis'
+on:
+  push:
+    branches:
+      - family/ripple
```

Comment:
Could you please change to `family/ripple-*`?
In accordance with proposal here: https://github.com/LedgerHQ/ledger-live-common/pull/1771

Also while I do see the benefit of the `family` tag to have dedicated bots, it doesn't really fit with the official tags in our proposed gitflow, cf. https://docs.google.com/presentation/d/1tgb2ikDaelddjglTR94Rq3h8-wBExeoMqm9iAWNstg0/edit#slide=id.g10d93822343_1_0
Maybe we should discuss with team so that everyone is on the same page on how to use that `family` tag.

Reply:
I don't think it's a good idea actually. This will make bot runs in parallel when 2 person work on 2 diff things and will create race conditions.

However, we can do this as soon as we figure out a way to "lock" (fail if running) on a given seed